### ISSUE TITLE
Crash When Selecting a "topic" from search results, subscribed channels

### ISSUE THREAD

**Issue by esotericDisciple** [TeamNewPipe/NewPipe#4776]
(Submitted using the FastHub-Libre app)

Step 1: Search for "aka to blue" (or import a newpipe-subscriptions file with "AKA TO BLUE - Topic" already added)

Step 2: Tap on the supposed "channel:" "AKA TO BLUE - Topic"

Step 2b: ...Having the "channel" in subscriptions also causes issues in feed syncing (EG: the feed error to pop up no matter where one goes in the app, as mentioned in the last error), but doesn't prevent feed sync from finising...

Crash log:

## Exception
* __User Action:__ requested channel
* __Request:__ https://www.youtube.com/channel/UCDuuCwdpsewH4MMSyy6XxOQ
* __Content Country:__ US
* __Content Language:__ en
* __App Language:__ en
* __Service:__ YouTube
* __Version:__ 0.20.2
* __OS:__ Linux samsung/beyond2qlteue/beyond2q:10/QP1A.190711.020/G975U1UEU4ETH8:user/release-keys 10 - 29
<details><summary><b>Crash log </b></summary><p>

```
org.schabi.newpipe.extractor.exceptions.ParsingException: Could not get channel id
	at org.schabi.newpipe.extractor.services.youtube.extractors.YoutubeChannelExtractor.getId(YoutubeChannelExtractor.java:133)
	at org.schabi.newpipe.extractor.channel.ChannelInfo.getInfo(ChannelInfo.java:60)
	at org.schabi.newpipe.extractor.channel.ChannelInfo.getInfo(ChannelInfo.java:48)
	at org.schabi.newpipe.util.ExtractorHelper.lambda$getChannelInfo$4(ExtractorHelper.java:124)
	at org.schabi.newpipe.util.-$$Lambda$ExtractorHelper$u5W7VszTe8AoEexIsFM9huQfbkM.call(Unknown Source:4)
	at io.reactivex.internal.operators.single.SingleFromCallable.subscribeActual(SingleFromCallable.java:44)
	at io.reactivex.Single.subscribe(Single.java:3666)
	at io.reactivex.internal.operators.single.SingleDoOnSuccess.subscribeActual(SingleDoOnSuccess.java:35)
	at io.reactivex.Single.subscribe(Single.java:3666)
	at io.reactivex.internal.operators.maybe.MaybeFromSingle.subscribeActual(MaybeFromSingle.java:41)
	at io.reactivex.Maybe.subscribe(Maybe.java:4290)
	at io.reactivex.internal.operators.maybe.MaybeConcatArray$ConcatMaybeObserver.drain(MaybeConcatArray.java:153)
	at io.reactivex.internal.operators.maybe.MaybeConcatArray$ConcatMaybeObserver.request(MaybeConcatArray.java:78)
	at io.reactivex.internal.operators.flowable.FlowableElementAtMaybe$ElementAtSubscriber.onSubscribe(FlowableElementAtMaybe.java:66)
	at io.reactivex.internal.operators.maybe.MaybeConcatArray.subscribeActual(MaybeConcatArray.java:42)
	at io.reactivex.Flowable.subscribe(Flowable.java:14935)
	at io.reactivex.internal.operators.flowable.FlowableElementAtMaybe.subscribeActual(FlowableElementAtMaybe.java:36)
	at io.reactivex.Maybe.subscribe(Maybe.java:4290)
	at io.reactivex.internal.operators.maybe.MaybeToSingle.subscribeActual(MaybeToSingle.java:46)
	at io.reactivex.Single.subscribe(Single.java:3666)
	at io.reactivex.internal.operators.single.SingleSubscribeOn$SubscribeOnObserver.run(SingleSubscribeOn.java:89)
	at io.reactivex.Scheduler$DisposeTask.run(Scheduler.java:578)
	at io.reactivex.internal.schedulers.ScheduledRunnable.run(ScheduledRunnable.java:66)
	at io.reactivex.internal.schedulers.ScheduledRunnable.call(ScheduledRunnable.java:57)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:301)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
	at java.lang.Thread.run(Thread.java:919)

```
</details>
<hr>

**Comment by FarahOshaibi** [comment]
json file must be tested correctly to get channel id because it might be incorrect that is why exception is occurring.

**Comment by Stypox** [comment]
Fixed in 0.20.3 (still RC)
